airbyte-integrations/connectors/source-mongodb-v2/build.gradle

```diff
@@ -58,7 +58,12 @@ dependencies {
     dataGeneratorImplementation libs.kotlinx.cli.jvm
     dataGeneratorImplementation 'org.yaml:snakeyaml:2.2'
 
-    debeziumTestImplementation libs.bundles.debezium.bundle
+    debeziumTestImplementation libs.debezium.api
+    debeziumTestImplementation libs.debezium.embedded
+    debeziumTestImplementation libs.debezium.sqlserver
+    debeziumTestImplementation libs.debezium.mysql
+    debeziumTestImplementation libs.debezium.postgres
+    debeziumTestImplementation libs.debezium.mongodb
```

Comment:
Any chance we can remove the sqlserver+mysql+postgres deps here? We're likely to forget to do this later on.

Reply:
it's on my list, but it's not as straightforward as it should be. I'm looking at it now, along with removing all the db-specific imports from the CDK.